packages/content-source-maps/package.json

```diff
@@ -47,11 +43,8 @@
     "vitest": "^1.2.2"
   },
   "devDependencies": {
-    "@swc/cli": "0.1.63",
-    "@swc/core": "1.3.100",
-    "@swc/jest": "0.2.29",
-    "@types/jest": "29.5.11",
-    "jest": "29.7.0",
-    "typescript": "5.1.6"
+    "chokidar": "3.6.0",
+    "typescript": "5.1.6",
+    "vite-plugin-dts": "2.3.0"
```

Comment:
vite, vitest & typescript should be then in the root package.json and also removed on the live-preview-sdk, potentially same for eslint if not already done